vllm/worker/model_runner.py

```diff
@@ -763,7 +753,6 @@ def _use_captured_graph(self,
                             max_decode_seq_len: int,
                             max_encoder_seq_len: int = 0) -> bool:
         return (decode_only and not self.runner.model_config.enforce_eager
-                and batch_size <= _BATCH_SIZES_TO_CAPTURE[-1]
                 and max_decode_seq_len <= self.runner.max_seq_len_to_capture
                 and max_encoder_seq_len <= self.runner.max_seq_len_to_capture
                 and batch_size <= self.runner.max_batchsize_to_capture)
```

Comment:
`self.runner.max_batchsize_to_capture <= _BATCH_SIZES_TO_CAPTURE[-1]`

Reply:
Sorry, can you please explain why we can skip this line?

Reply:
the condition `self.runner.max_batchsize_to_capture <= _BATCH_SIZES_TO_CAPTURE[-1]` always holds, and if `batch_size <= self.runner.max_batchsize_to_capture` , `batch_size <= _BATCH_SIZES_TO_CAPTURE[-1]` naturally holds.

this is just redundant code.

Reply:
Got it. Thanks for the explanation. Didn't see L769.